lib/optimizely/config_manager/http_project_config_manager.rb

```diff
@@ -148,17 +150,16 @@ def request_config
         "Fetching datafile from #{@datafile_url}"
       )
       begin
-        headers = {
-          'Content-Type' => 'application/json'
-        }
+        uri = URI.parse(@datafile_url)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.read_timeout = Helpers::Constants::CONFIG_MANAGER['REQUEST_TIMEOUT']
+        http.use_ssl = true
```

Comment:
this is not true at all, datafile server can be without `ssl` if using custom URL.

Reply:
nice catch